payment_cycle/migrations/0001_initial.py

```diff
@@ -0,0 +1,69 @@
+                ('user_created', models.ForeignKey(db_column='UserCreatedUUID', on_delete=django.db.models.deletion.DO_NOTHING, related_name='paymentcycle_user_created', to=settings.AUTH_USER_MODEL)),
+                ('user_updated', models.ForeignKey(db_column='UserUpdatedUUID', on_delete=django.db.models.deletion.DO_NOTHING, related_name='paymentcycle_user_updated', to=settings.AUTH_USER_MODEL)),
+            ],
+            options={
+                'abstract': False,
+            },
+            bases=(dirtyfields.dirtyfields.DirtyFieldsMixin, models.Model),
+        ),
+        migrations.CreateModel(
+            name='HistoricalPaymentCycle',
+            fields=[
+                ('id', models.UUIDField(db_column='UUID', db_index=True, default=None, editable=False)),
+                ('is_deleted', models.BooleanField(db_column='isDeleted', default=False)),
+                ('json_ext', models.JSONField(blank=True, db_column='Json_ext', null=True)),
+                ('date_created', core.fields.DateTimeField(db_column='DateCreated', null=True)),
+                ('date_updated', core.fields.DateTimeField(db_column='DateUpdated', null=True)),
+                ('version', models.IntegerField(default=1)),
+                ('run_year', models.IntegerField()),
+                ('run_month', models.SmallIntegerField()),
+                ('history_id', models.AutoField(primary_key=True, serialize=False)),
+                ('history_date', models.DateTimeField(db_index=True)),
+                ('history_change_reason', models.CharField(max_length=100, null=True)),
+                ('history_type', models.CharField(choices=[('+', 'Created'), ('~', 'Changed'), ('-', 'Deleted')], max_length=1)),
+                ('history_user', models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='+', to=settings.AUTH_USER_MODEL)),
+                ('type', models.ForeignKey(blank=True, db_constraint=False, null=True, on_delete=django.db.models.deletion.DO_NOTHING, related_name='+', to='contenttypes.contenttype')),
+                ('user_created', models.ForeignKey(blank=True, db_column='UserCreatedUUID', db_constraint=False, null=True, on_delete=django.db.models.deletion.DO_NOTHING, related_name='+', to=settings.AUTH_USER_MODEL)),
+                ('user_updated', models.ForeignKey(blank=True, db_column='UserUpdatedUUID', db_constraint=False, null=True, on_delete=django.db.models.deletion.DO_NOTHING, related_name='+', to=settings.AUTH_USER_MODEL)),
+            ],
+            options={
+                'verbose_name': 'historical payment cycle',
+                'verbose_name_plural': 'historical payment cycles',
+                'ordering': ('-history_date', '-history_id'),
+                'get_latest_by': ('history_date', 'history_id'),
+            },
+            bases=(simple_history.models.HistoricalChanges, models.Model),
+        ),
+    ]
```

Comment:
Do we have any perms for displaying payment cycle and run payment cycle? If yes - we have to add those roles for admin in migration too.